src/main/java/picard/util/BaitDesigner.java

```diff
@@ -489,6 +497,40 @@ protected int doWork() {
             }
         }
 
+        if (AUX_BAITS != null) {
+            final IntervalList auxBaits = IntervalList.fromFile(AUX_BAITS);
+            SequenceUtil.assertSequenceDictionariesEqual(auxBaits.getHeader().getSequenceDictionary(),
+                    baits.getHeader().getSequenceDictionary());
+
+            final ReferenceSequenceFileWalker walker = new ReferenceSequenceFileWalker(REFERENCE_SEQUENCE);
+            ReferenceSequence reference = null;
+            final SAMSequenceDictionary dict = auxBaits.getHeader().getSequenceDictionary();
+            for (final Interval interval : auxBaits.getIntervals()) {
+                final int contigIndex = dict.getSequenceIndex(interval.getContig());
+                if (reference == null || contigIndex != reference.getContigIndex()) {
+                    reference = walker.get(contigIndex);
+                }
+                final Bait bait = new Bait(interval.getContig(),
+                        interval.getStart(),
+                        interval.getEnd(),
+                        interval.isNegativeStrand(),
+                        interval.getName()
+                );
+                bait.addBases(reference, DESIGN_ON_TARGET_STRAND);
```

Comment:
this is a little strange, since AUX_BAIT might have been designed on both strands, and if DESIGN_ON_TARGET_STRAND is false the code will ignore that information

Reply:
given that the interval already has a positive/negative strand information, shouldn't you always use "true" in the second argument?